db/schema.rb

```diff
@@ -29,6 +29,8 @@
     t.datetime "created_at",   null: false
     t.datetime "updated_at",   null: false
     t.integer  "affiliate_id"
+    t.string   "phone_number"
+    t.date     "dob"
```

Comment:
@xingyif but I have made the name change you requested :)   
it is now date_of_birth